test/chairs.py

```diff
@@ -0,0 +1,323 @@
+#!/usr/bin/env python3
+"""
+Test chars and chair-mp mapping metadata
+"""
+import pandas as pd
+import unittest
+import warnings
+
+
+
+
+class ChairHog(Warning):
+
+    def __init__(self, m):
+        self.message = f"There following MPs sit in two chairs in {m}:."
+
+
+    def __str__(self):
+        return self.message
+
+
+
+
+class ChairInWrongTid(Warning):
+
+    def __init__(self, m):
+        self.message = f"There is {m}."
+
+
+    def __str__(self):
+        return self.message
+
+
+
+
+class ChairMissingFromRange(Warning):
+
+    def __init__(self, m):
+        self.message = f" in {m} is missing."
+
+
+    def __str__(self):
+        return self.message
+
+
+
+
+class ChairOOR(Warning):
```

Comment:
What is OOR? please spell out,

Reply:
OutOfRange -- is it sufficient for you  if I add a comment to that effect, or you want me to change the whole class name?

Reply:
Let's try to have as clear class names as possible. So I would call it ChairOutOfRange, as you suggest.

Reply:
adc10672d3e165e0a76d183803d5cba7e3561f72